ml/src/pipeline/ecs_gpu_worker.py

```
@@ -0,0 +1,126 @@
+        s3.upload_file(output_path, output_bucket, output_key)
+    except Exception as e:
+        print(f"ERROR: Failed to upload to S3: {e}", file=sys.stderr)
+        sys.exit(1)
+
+    success_count = sum(1 for r in results if r.get("embedding") is not None)
+    print(f"SUCCESS: {success_count}/{len(results)} records processed")
+    sys.exit(0)
+
+
+if __name__ == "__main__":
+    main()
```

Ruff (0.15.12)

[warning] No newline at end of file

Add trailing newline

(W292)